haas/ext/auth/database.py

```diff
@@ -137,6 +137,12 @@ def authenticate(self):
         else:
             return False
 
+    def get_user(self):
+        if hasattr(local, 'auth') and local.auth:
```

Comment:
@gsilvis after your +1 I changed this line to also check if `local.auth` is not None (in the case of a failed authentication local.auth would be set to None therefore the first condition isn't enough and the server returned 500 since it then tried to access local.auth.label. Caught this bug during testing.